clodius/cli/aggregate.py

```diff
@@ -263,7 +267,7 @@ def line_to_dict(line):
             raise(ValueError(error_str))
         entries = [line_to_dict(first_line)]
 
-    entries += [line_to_dict(line.strip()) for line in f]
+    entries += [line_to_dict(line.strip()) for line in f if len(line.strip())]
```

Comment:
Since `line.strip()` is repeated, it might be more clear to pull it out? And I don't think the `len()` is needed?
```python
entries += [line_to_dict(line) for line in [line.strip() for line f] if line]
```

Reply:
Good point.